src/utils/bootstrap/bootstrap.service.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { BootstrapService } from './bootstrap.service';
+
+describe('BootstrapService', () => {
```

Comment:
Can you explain the purpose of this class?

Reply:
Test harness auto-generated by nestjs.